cpp/src/slice2cs/CsUtil.cpp

```diff
@@ -298,7 +243,8 @@ string
 Slice::CsGenerator::resultStructName(const string& className, const string& opName, bool marshaledResult)
 {
     ostringstream s;
-    s << className << "_" << IceInternal::toUpper(opName.substr(0, 1)) << opName.substr(1)
+    string fixedOpName = addPrefixToIdentifier("", opName); // Strip any leading '@' from the name.
```

Comment:
I don't like using empty prefix as a mean of removing `@`

Reply:
I agree!

Reply:
Use my helper function.

Reply:
I used your helper function.
But named it `removeEscapePrefix` instead of `removeAtPrefix` because the latter sounds like it's removing something at a position, which it isn't.